.github/workflows/ci.yml

```diff
@@ -69,3 +69,30 @@ jobs:
           bundler-cache: true
       - name: Run rubocop
         run: bundle exec rake lint
+
+  functional:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: ["2.0", "ruby"]
```

Comment:
🗒️ `"ruby"` in this context means the latest stable version of Ruby. As of this writing, it is 3.3.3. Using this keyword means the functional tests will always run against the latest version of Ruby, without us having to manually update the workflow.